bungeecord/src/main/java/io/github/retrooper/packetevents/bstats/Metrics.java

```diff
@@ -28,8 +28,12 @@
  * bStats collects some data for plugin authors.
  * <p>
  * Check out https://bStats.org/ to learn more about bStats!
+ * <p>
+ * @deprecated This class is deprecated and will be removed in future versions.
+ * Please use 'io.github.retrooper.packetevents.bstats.bungeecord' instead.
```

Comment:
@retrooper is this api or internal?
If it's internal, this should be rephrased to "Please shade your own bstats copy instead.".

Reply:
I am pretty sure this is API. Retrooper once told me to use this class, instead of shading my own.

Reply:
We can make an exception for this one. I probably only told you that you can use it. Let's remove the old class.

Reply:
@booky10 @Bram1903

Reply:
@retrooper, so do you want me to remove the classes I now marked as deprecated?

Reply:
Yeah.